database/index.js

```diff
@@ -0,0 +1,56 @@
+const Sequelize = require('sequelize');
+const seed = require('./seed.js');
+
+console.log(seed.seed)
+
+// Option 1: Passing parameters separately
+const sequelize = new Sequelize('yelpPhotoGallery', 'root', '', {
+  host: 'localhost',
+  dialect: 'mysql',
+  define: {
+    // The `timestamps` field specify whether or not the `createdAt` and `updatedAt` fields will be created.
+    // This was true by default, but now is false by default
+    timestamps: false
+  }
+});
+
+sequelize
+  .authenticate()
+  .then(() => {
+    console.log('Connection has been established successfully.');
+  })
+  .catch(err => {
+    console.error('Unable to connect to the database:', err);
+  });
+
+  const Users = sequelize.define('users', {
+    name: { type: Sequelize.STRING },
+    stars: { type: Sequelize.INTEGER },
+    reviews: { type: Sequelize.INTEGER }
+  });
+
+  const Businesses = sequelize.define('businesses', {
+    name: { type: Sequelize.STRING }
+  });
+
+  const Images = sequelize.define('images', {
+    image_url: { type: Sequelize.STRING },
+    helpfull_count: { type: Sequelize.INTEGER },
+    not_helpfull_Count: { type: Sequelize.INTEGER },
+    description: { type: Sequelize.STRING },
+    date: { type: Sequelize.DATEONLY },
+    business_id: { type: Sequelize.INTEGER}, //This needs to be foreign key refference
+    users_id: { type: Sequelize.INTEGER } //This needs to be foreign key refference
+  });
+
+
+  Users.sync({ force: true }).then(() => {
```

Comment:
consider keeping your database schema/model setup in a separate file from your seeding function calls. From the way it is written here, every time you require this file, Sequalize would be resyncing your database and reseeding your tables with your sample data rather than only when you run your npm script specifying to do so.